### DISCUSSION TITLE
Bot not working after the deploy i am using google  cloud console and heroku app

### DISCUSSION THREAD

**Discussion by samhdbackup12** [anasty17/mirror-leech-telegram-bot#946] [Q&A]
Can anyone tell me why the bot is not working after  following all the steps in the video. The workflow is completed successfully but when i give command to the bot the bot is not responding.

**Comment by anasty17** [comment]
Logs!

**Reply by anasty17** [reply]
What r u doing! Deploy heroku branch not master!

**Reply by anasty17** [reply]
Readme! No need to ask anything!
